Sources/KSCrashFilters/include/KSCrashReportFilterBasic.h

```diff
@@ -56,15 +61,37 @@
  *                    Each "filter" can be id<KSCrashReportFilter> or an NSArray
  *                    of filters (which gets wrapped in a pipeline filter).
  */
-+ (KSCrashReportFilterCombine*) filterWithFiltersAndKeys:(id) firstFilter, ... NS_REQUIRES_NIL_TERMINATION;
++ (instancetype) filterWithFiltersAndKeys:(nullable id) firstFilter, ... NS_REQUIRES_NIL_TERMINATION;
```

Comment:
Should we drop nil-termenated initialization? Not sure if it's used anywhere, but maybe it's just me

Reply:
It's not visible for C, but not for Swift, that's why a implemented with `NSArray` side-by-side.